src/main/java/io/opentracing/contrib/jdbc/TracingDriver.java

```diff
@@ -34,6 +34,25 @@
 
   private static final String WITH_ACTIVE_SPAN_ONLY = TRACE_WITH_ACTIVE_SPAN_ONLY + "=true";
 
+  public static final String MYSQL_DRIVER      = "com.mysql.jdbc.Driver";
```

Comment:
I think we don't need all these driver class variables, see my comment in the usage of them.